### ISSUE TITLE
Why make backbone a hard dependency 

### ISSUE THREAD

**Issue by digitalgym** [bittorrenttorque/btapp#18]
I would like to library which is not dependent on a specific MVC MVW framework makes integration easier.

**Comment by ryan-lane** [comment]
Indeed. I have an angularjs project I'd really like to use this in...

**Comment by galeksandrp** [comment]
http://schizoduckie.github.io/DuckieTorrent/

> I ♥ necroposts.
